### ISSUE TITLE
Los Problem with Multiples Npcs.

### ISSUE THREAD

**Issue by Rorschach91** [azerothcore/azerothcore-wotlk#19697]
Bug Triager Notes:

Original CC Report: https://github.com/chromiecraft/chromiecraft/issues/7290#issue-2473649703
Original CC Report: https://github.com/chromiecraft/chromiecraft/issues/7294#issue-2474112956

I checked these mobs on CC standard and PTR. This issue is only present on CC standard (on PTR npcs work well).
I only copy one issue, but I link both reports that talk about the same problem but with different npcs.

### What client do you play on?

enGB

### Faction

Alliance

### Content Phase:

70

### Current Behaviour

When you pull some of mobs and run away (LOS), they do not follow, but keep watching and giving you combat problems until you run to them to actually aggro them.

![WoWScrnShot_080724_073315](https://github.com/user-attachments/assets/ef9a5c8d-866b-4546-8dd3-ddedda3b7d2a)
![WoWScrnShot_081124_164956](https://github.com/user-attachments/assets/4ee9a0dd-8b0a-4cfe-9f0d-b95adece0681)
![WoWScrnShot_081024_153850](https://github.com/user-attachments/assets/28cb9219-9358-44e9-a98f-3589044ebc75)

### Expected Blizzlike Behaviour

Should follow you 

### Source

-

### Steps to reproduce the problem

Not sure, but here are screenshots of multiple problems within different days, situations and charaters.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/23f58016d078e922ebbc378f28223d6d0a19693d

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by heyitsbench** [comment]
Add [Cabal Abjurists](https://wowgaming.altervista.org/aowow/?npc=21660) to the list.

**Comment by heyitsbench** [comment]
> probably map issue

I don't believe so, these issues seem to stem from https://github.com/azerothcore/azerothcore-wotlk/commit/8f127f9e2135a70cdb5573d052f50304d1b3875c (might also be a duplicate of https://github.com/azerothcore/azerothcore-wotlk/issues/18779), but it seems to be solved for individual creatures by adding a flag to smart AI actions iirc.

**Comment by Tereneckla** [comment]
in any case, Cabal Abjurist still has the combat movement hackfix SAI

**Comment by Rorschach91** [comment]
Re-tested after PR: https://github.com/azerothcore/azerothcore-wotlk/pull/23005

Twilight's Hammer Torturers, Bloodmaul Geomancers and Bristleback Geomancer seem to be working well.
For Cabal Abjurists, Sunfury Astromancers and Magister I reworked their SmartAI and they are working well now.

I close this report as completed.
